default/scripting/species/common/research.macros

```diff
@@ -1,5 +1,8 @@
+NO_RESEARCH
+'''[[DESCRIPTION_EFFECTSGROUP_MACRO(NO_RESEARCH_DESC)]]'''
+
```

Comment:
Grooming: moved to top of file NO_RESEARCH macro to match the industry.macros ordering.